dynawo/sources/API/DYD/DYDModelTemplateImpl.cpp

```diff
@@ -95,20 +96,7 @@ ModelTemplate::Impl::addUnitDynamicModel(const shared_ptr<UnitDynamicModel>& mod
 ModelTemplate&
 ModelTemplate::Impl::addConnect(const string& model1, const string& var1,
         const string& model2, const string& var2) {
-  if (unitDynamicModelsMap_.find(model1) == unitDynamicModelsMap_.end())
-    throw DYNError(DYN::Error::API, ConnectorNotPartofModel, model1, model2, getId());
-  if (unitDynamicModelsMap_.find(model2) == unitDynamicModelsMap_.end())
-    throw DYNError(DYN::Error::API, ConnectorNotPartofModel, model2, model1, getId());
-
-  string pc_first = model1 + '_' + var1;
-  string pc_second = model2 + '_' + var2;
-  // To build the connector Id, sort the string so as 1st_Model_ID is smaller than 2nd_Model_ID. EX: ID_1 < ID_2
-  string pc_Id;
-  if (pc_first < pc_second)
-    pc_Id = pc_first + '_' + pc_second;
-  else
-    pc_Id = pc_second + '_' + pc_first;
-
+  string pc_Id = getConnectionId(model1, var1, model2, var2, getId(), unitDynamicModelsMap_);
```

Comment:
what is the meaning of pc ?

Reply:
absolutely no idea. I kept the name but I can change it if necessary

Reply:
I saw it is also used in the dictionnary so maybe if we don't someone else might not know... if it has a real meaning...
there is the same with ic below, I presumed init connect but not sure also

Reply:
According to Adrien it might be "part" of the model, as it is a connection within the model

Reply:
It means Pin Connection

Reply:
Solved